grails-app/resourceMappers/CorsResourceMapper.groovy

```diff
@@ -1,9 +1,4 @@
-import org.grails.plugin.resource.mapper.MapperPhase
-
-
 class CorsResourceMapper {
-	static phase = MapperPhase.ALTERNATEREPRESENTATION
```

Comment:
MapperPhase class comes from Grails resources plugin (which is now removed). Resources plugin handles static assets (images/etc). Since there are no more static resources this can be removed. ALTERNATEPRSENTATION refers to gzip.